diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/MagicNumberRule.kt

```diff
@@ -102,13 +102,20 @@ class MagicNumberRule(configRules: List<RulesConfig>) : DiktatRule(
         /**
          * Check if string is number
          */
+        // || ((this.last().uppercase() == "L" || this.last().uppercase() == "U") && this.substring(0, this.lastIndex-1).isNumber())
```

Comment:
Probably we can finally improve our `commented_out_code` rule, to ignore leading whitespaces and make trigger on such cases 😐